plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -226,6 +226,33 @@ export async function createRouter(
     res.json(responses.filter(r => accessibleEntityRefs.includes(r.userRef)));
   });
 
+  router.get('/app-ratings', async (_, res) => {
+    const ratings = await dbHandler.getAppRatings();
+    res.status(200).json(ratings);
+  });
+
+  router.post('/app-ratings', async (req, res) => {
+    const user = await identity.getIdentity({ request: req });
+    const rating = req.body.rating;
+    if (!rating) {
+      res.status(400).end();
+      return;
+    }
+
+    const ratingNum = Number(rating);
+    if (isNaN(ratingNum)) {
```

Comment:
I think we'd want to make sure it's in a given range here too right? and probably an integer.